### PR TITLE
kernel upgraded and gpd-fan removed

### PR DESCRIPTION
**Pull request by cawilliamson**
There's a few changes here:

- Updated to latest PKGBUILD for linux-jwrdegoede from [njkli/gpd-pocket](https://github.com/njkli/gpd-pocket/blob/master/linux-jwrdegoede/PKGBUILD)
- Added fix (PR'd to njkli/gpd-pocket - link below) for booting from mmc without any system changes
- Removed gpd-fan since this is now built in to 4.15+ kernels

Also removed fbcon=rotate:1 - this has been in the hans kernel for a while now and I believe it's even in the mainline kernel to auto detect GPD Pocket and flip the screen.

Related:
https://github.com/njkli/gpd-pocket/pull/14

### CODE REVIEW

File: archlive/airootfs/usr/bin/gpd-setup [hunk 1]
```diff
@@ -32,9 +32,7 @@ add_repo() {
 
 set_settings() {
         mkdir -p ${volume}/etc/default && cp /etc/default/tlp ${volume}/etc/default/tlp && \
-        mkdir -p ${volume}/etc/default && cp /etc/default/gpd-fan ${volume}/etc/default/gpd-fan && \
         echo "Successfully defined default settings" || \
-        echo "Unable to defined default settings for tlp and gpd-fan. Please see archwiki page on GPD Pocket."
```
Comment (sigboe) [inline archlive/airootfs/usr/bin/gpd-setup]: Removing this line breaks the try catch that displays the error message. Its better to just remove `and gpd-fan`

File: packages/linux-jwrdegoede/PKGBUILD [hunk 2]
```diff
@@ -225,7 +224,6 @@ CONFIG_FB_VIA_DIRECT_PROCFS y
 CONFIG_VBOXGUEST n
 CONFIG_DRM_VBOXVIDEO n
 CONFIG_VBOXSF_FS n
-CONFIG_MTD_PHRAM=m
```
Comment (sigboe) [inline packages/linux-jwrdegoede/PKGBUILD]: @cawilliamson Removing this line breaks booting into live environment.

@joshskidmore Can you review the rest of the changes this file?

**Comment by cawilliamson** [comment]
Pushed CONFIG_MTD_PHRAM=m back upstream to https://github.com/njkli/gpd-pocket/pull/15 - thanks for the heads on on that!

Clearly didn't have my coffee before I edited the gpd-setup script though - sorry about that!

Interestingly - my kernel doesn't have CONFIG_MTD_PHRAM enabled right now and seems to be running fine but will rebuild with that enabled.

**Comment by joshskidmore** [comment]
`CONFIG_MTD_PHRAM` is only needed for booting with the USB installer; it's not needed for installed kernel itself.

**Comment by sigboe** [comment]
Want to push the new ISO as soon as possible, so I merged this into a new branch where I can mess around easier. I'll get around to giving you write access too the repo soon @cawilliamson 

I made the new branch because as soon as the `CONFIG_MTD_PHRAM` binary is posted to njkli as a binary, I can drop the whole pkgbuild repo part of this github site, and only focus on the ArchISO stuff.  Then updates can be pushed much quicker as the arch repo part is a bit heavy to change all the time.

**Comment by sigboe** [comment]
@joshskidmore Do you have the signing key or need the key to push new binaries in njkli?

**Pull request merged**